forge-gui/res/cardsfolder/c/ceta_sanctuary.txt

```diff
@@ -1,14 +1,12 @@
 Name:Ceta Sanctuary
 ManaCost:2 U
 Types:Enchantment
-T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Permanent.YouCtrl+Red,Permanent.YouCtrl+Green | Execute$ TrigDraw | TriggerDescription$ At the beginning of your upkeep, if you control a red or green permanent, draw a card, then discard a card. If you control a red permanent and a green permanent, instead draw two cards, then discard a card.
-SVar:TrigDraw:DB$ Draw | NumCards$ C | SubAbility$ DBDiscard
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Permanent.YouCtrl+Red | IsPresent2$ Permanent.YouCtrl+Green | PresentCompare2$ EQ0 | Execute$ TrigDraw | TriggerDescription$ At the beginning of your upkeep, if you control a red or green permanent, draw a card, then discard a card. If you control a red permanent and a green permanent, instead draw two cards, then discard a card.
```

Comment:
Hmn, I'm not sure saving the SVars by splitting the trigger into three is a good improvement :/

Reply:
I also wasn't sure that was "better" 😅